### PR TITLE
tailscale-ssh: add page

### PR DESCRIPTION
**Pull request by mfrw**
Closes: #8150

Signed-off-by: Muhammad Falak R Wani <falakreyaz@gmail.com>

- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

**Review by einverne — COMMENTED** [review]
It looks like tailscale ssh is a sub command of tailscale, and even there is no `tailscale ssh` sub command, should we merge these examples into `tailscale up`?

**Comment by navarroaxel** [comment]
@einverne `tailscale up` has a lot of options, maybe the turn on/off the SSH should be in this page.

**Comment by mfrw** [comment]
> @einverne `tailscale up` has a lot of options, maybe the turn on/off the SSH should be in this page.

It already had 8 examples and all of them seemed important; which is the reason I added these here.

**Comment by sbrl** [comment]
> and even there is no tailscale ssh sub command


I'm confused. In this page, there's the example `tailscale ssh {{user}}@{{host}}`. Does the `ssh` subcommand exist, or not?

Either way, the title of a page *must* match the command name. For example, `awk.md` is the page for `awk`, `docker-ps.md` is the page for `docker ps`, and so on. The only exception is special case disambiguation pages such as `cgroups` for things users will commonly type.

**Comment by mfrw** [comment]
> > and even there is no tailscale ssh sub command
> 
> I'm confused. In this page, there's the example `tailscale ssh {{user}}@{{host}}`. Does the `ssh` subcommand exist, or not?
> 
> Either way, the title of a page _must_ match the command name. For example, `awk.md` is the page for `awk`, `docker-ps.md` is the page for `docker ps`, and so on. The only exception is special case disambiguation pages such as `cgroups` for things users will commonly type.
> 
> To this end, if the `tailscale ssh` subcommand does not exist, I'm not sure we can accept this. The art of writing a tldr page is distilling the most useful knowledge down into a quick reference for new and experienced users alike - not providing comprehensive documentation.

`tailscale ssh` subcommand exists :)
The benefit of `tailscale ssh` is `password-less/key-less access` to host machines.
Ideally the options to `enable/disable` ssh should be in the `tailscale up` page, but that page already has 8 examples which are all pretty usefull.
Since `enable/disable` seemed helpful and make more sense here, it is the reason I added them here :)

image

**Comment by sbrl** [comment]
Thanks for clarifying, @mfrw! In this case, are the `sudo tailscale up` commands required for using the `tailscale ssh` command?

**Comment by navarroaxel** [comment]
> Thanks for clarifying, @mfrw! In this case, are the `sudo tailscale up` commands required for using the `tailscale ssh` command?

@sbrl The `--ssh=true` is required to enable this SSH feature.

**Comment by mfrw** [comment]
> Thanks for clarifying, @mfrw! In this case, are the `sudo tailscale up` commands required for using the `tailscale ssh` command?

Yep, good catch. we do need `sudo`.
Fixing :)

File: pages/common/tailscale-ssh.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+- Advertise SSH on the host:
+
+`sudo tailscale up --ssh=true`
+
+- Disable SSH on the host:
+
+`sudo tailscale up --ssh=false`
+
+- SSH to a host which has Tailscale-SSH enabled:
```
Comment (EmilySeville7cfg) [inline pages/common/tailscale-ssh.md:14]:
```suggestion
- SSH to a specific host which has Tailscale-SSH enabled:
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

**Review by sbrl — APPROVED** [review]

**Review by EmilySeville7cfg — APPROVED** [review]

File: pages/common/tailscale-ssh.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# tailscale ssh
+
+> SSH to a Tailscale machine (Linux Only).
+> More information: <https://tailscale.com/kb/1193/tailscale-ssh>.
+
+- Advertise/Disable SSH on the host:
```
[the remark this answers is not in the document]
Comment (mfrw) [reply pages/common/tailscale-ssh.md:6]: I was of the opinion to keep it `/` and keep it abstract :)

**Pull request merged**